metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -275,15 +275,40 @@ object MetalsEnrichments
         workspace.resolve(Directories.readonly).toNIO
       )
 
+    def isInDependenciesDirectory(workspace: AbsolutePath): Boolean =
+      path.toNIO.startsWith(
+        workspace.resolve(Directories.dependencies).toNIO
+      )
+
+    def jarPath: Option[AbsolutePath] = {
+      val filesystem = path.toNIO.getFileSystem()
+      if (filesystem.provider().getScheme().equals("jar")) {
+        Some(
+          AbsolutePath(
+            Paths.get(filesystem.toString)
+          )
+        )
+      } else {
+        None
+      }
+    }
+
     /**
      * Writes zip file contents to disk under $workspace/.metals/readonly.
+     *
+     * In case if path is jar then target directory is $workspace/.metals/readonly/$jarName
```

Comment:
So I'm a bit confused by this. So what is the difference between:

```
metals/readonly/dependencies/scala-library_2.13.3-sources.jar/scala/Predef.scala
```
and
```
metals/readonly/scala-library_2.13.3-sources.jar/scala/Predef.scala
```

Reply:
I forgot to add `/dependencies` in the comment. Thx!